besu-plugins/state-recovery/test-cases/src/integrationTest/kotlin/linea/staterecovery/datafetching/SubmissionsFetchingTaskIntTest.kt

```diff
@@ -167,7 +167,7 @@ class SubmissionsFetchingTaskIntTest {
   @Test
   fun `should stop fetching submissions once debugForceSyncStopBlockNumber is reached`() {
     val debugForceSyncStopBlockNumber =
-      aggregationsAndBlobs[aggregationsAndBlobs.size - 2].aggregation!!.endBlockNumber - 1UL
+      aggregationsAndBlobs[aggregationsAndBlobs.size - 1].aggregation!!.endBlockNumber - 1UL
```

Comment:
I don't really get this change, nor do I get why it was - 2 before. Can you explain?

Reply:
it is a product of limited data, because I have fewer aggregations in the test data, the extra one is out of bounds.

This still performs the same level of testing, in that aggregation submission pauses at the `debugForceSyncStopBlockNumber`